### ISSUE TITLE
ca-certificates was not found when building AMI (v0.10.0)

### ISSUE THREAD

**Issue by k-o-ta** [privacysandbox/protected-auction-key-value-service#18]
Hi,
When I run `production/packaging/aws/build_and_test`, I got an error and found an error log.

```
# less /path/to/fledge-key-value-service/docker-buildx-presubmit-BYfp.log
#1 [internal] load remote build context
#1 CACHED

#2 copy /context /
#2 CACHED

#3 [internal] load metadata for docker.io/library/ubuntu:20.04
#3 DONE 0.0s

#4 [1/4] FROM docker.io/library/ubuntu:20.04
#4 DONE 0.0s

#5 [2/4] COPY install_apps install_go.sh .pre-commit-config.yaml /scripts/
#5 CACHED

#6 [3/4] COPY gitconfig /etc
#6 CACHED

#7 [4/4] RUN    chmod 644 /etc/gitconfig &&   /usr/bin/env -v PRE_COMMIT_VENV_DIR=/usr/pre-commit-venv /scripts/install_apps &&   rm -rf /scripts
#7 0.230 setenv:   PRE_COMMIT_VENV_DIR=/usr/pre-commit-venv
#7 0.230 executing: /scripts/install_apps
#7 0.230    arg[0]= '/scripts/install_apps'
#7 0.612 Get:1 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]
#7 0.721 Get:2 http://archive.ubuntu.com/ubuntu focal InRelease [265 kB]
#7 1.449 Get:3 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [1056 kB]
#7 2.019 Get:4 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
#7 2.280 Get:5 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [2773 kB]
#7 2.323 Get:6 http://archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]
#7 2.625 Get:7 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages [1275 kB]
#7 2.634 Get:8 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 Packages [28.5 kB]
#7 2.636 Get:9 http://security.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [2345 kB]
#7 3.223 Get:10 http://archive.ubuntu.com/ubuntu focal/universe amd64 Packages [11.3 MB]
#7 4.290 Get:11 http://archive.ubuntu.com/ubuntu focal/multiverse amd64 Packages [177 kB]
#7 4.293 Get:12 http://archive.ubuntu.com/ubuntu focal/restricted amd64 Packages [33.4 kB]
#7 4.293 Get:13 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1351 kB]
#7 4.392 Get:14 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [31.2 kB]
#7 4.393 Get:15 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [2483 kB]
#7 4.780 Get:16 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [3252 kB]
#7 5.495 Get:17 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [28.6 kB]
#7 6.037 Get:18 http://archive.ubuntu.com/ubuntu focal-backports/main amd64 Packages [55.2 kB]
#7 6.331 Fetched 26.8 MB in 6s (4412 kB/s)
#7 6.331 Reading package lists...
#7 6.978 Reading package lists...
#7 7.596 Building dependency tree...
#7 7.745 Reading state information...
#7 7.889 E: Version '20211016ubuntu0.20.04.1' for 'ca-certificates' was not found
#7 ERROR: executor failed running [/bin/sh -c chmod 644 /etc/gitconfig &&   /usr/bin/env -v PRE_COMMIT_VENV_DIR=${PRE_COMMIT_VENV_DIR} /scripts/install_apps &&   rm -rf /scripts]: exit code: 100
------
 > [4/4] RUN    chmod 644 /etc/gitconfig &&   /usr/bin/env -v PRE_COMMIT_VENV_DIR=/usr/pre-commit-venv /scripts/install_apps &&   rm -rf /scripts:
#7 4.293 Get:12 http://archive.ubuntu.com/ubuntu focal/restricted amd64 Packages [33.4 kB]
#7 4.293 Get:13 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1351 kB]
#7 4.392 Get:14 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [31.2 kB]
#7 4.393 Get:15 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [2483 kB]
#7 4.780 Get:16 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [3252 kB]
#7 5.495 Get:17 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [28.6 kB]
#7 6.037 Get:18 http://archive.ubuntu.com/ubuntu focal-backports/main amd64 Packages [55.2 kB]

#7 7.745 Reading state information...
#7 7.889 E: Version '20211016ubuntu0.20.04.1' for 'ca-certificates' was not found
------
error: failed to solve: executor failed running [/bin/sh -c chmod 644 /etc/gitconfig &&   /usr/bin/env -v PRE_COMMIT_VENV_DIR=${PRE_COMMIT_VENV_DIR} /scripts/install_apps &&   rm -rf /scripts]: exit code: 100
```
There is a news that ca-certificates were updates.
https://ubuntu.com/security/notices/USN-6105-1

After [updating these files](https://github.com/k-o-ta/fledge-key-value-service/commit/bd6e1ad304c22c4d42f6627133e9702ad8bb7f2f), the build was succeeded.

**Comment by peiwenhu** [comment]
Thanks. Yes we noticed this too. it's understandable but a bit unfortunate that Ubuntu just upgraded it and removed the older version. we're working on handling this type of situation better and for this immediate issue our next release will include the necessary fixes.

**Comment by peiwenhu** [comment]
This should be addressed in the newer release 0.11.0
